andela/titan-dev-ah-frontend
# #163518658 Search & Filter Articles

#### What does this PR do?
Search Article by keyword
#### Description of Task to be completed?
- Convert HTML & CSS mock into React component
- Fetchs article and store them into Redux state
- Displies fetchec article 
- Test: component, reducers, and action creators
- Refactors app routing
#### How should this be manually tested?
- clone the repo and check out the following branch `ft-search-and-filtering-163518658 `
- run `yarn start` and navigate to / in your browser, type in a search query to be redirected to search results page
### Related Pivotal tracker ID
#163518658
